geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/ShowStackTraceDUnitTest.java

```diff
@@ -90,6 +90,20 @@ public void run() {
   public void testExportStacktrace() throws ClassNotFoundException, IOException {
     setupSystem();
 
+    // Test non txt extension file is allowed
+    File stacktracesFile = new File("allStackTraces.log");
```

Comment:
Files should be created in a TemporaryFolder. Please see TemporaryFolder on https://github.com/junit-team/junit4/wiki/Rules.

We have a version of TemporaryFolder which works in DUnit (DistributedTests). See org.apache.geode.test.junit.rules.serializable.SerializableTemporaryFolder.

The TemporaryFolder Rule automatically takes care of deleting files and directories after each test.